src/text/line.rs

```diff
@@ -475,26 +475,26 @@ impl<'a> Line<'a> {
 }
 
 impl<'a> IntoIterator for Line<'a> {
-    type Item = Span<'a>;
     type IntoIter = std::vec::IntoIter<Span<'a>>;
```

Comment:
The IntoIter types can use Self::Item for a small simplification. Not a blocker though.
```suggestion
    type IntoIter = std::vec::IntoIter<Self::Item>;
```

I'm not too worried about the type order here - it's odd (because it changes the order from the definition), but it's not something that would make anyone unable to understand the code.

Reply:
It's a bit funny given that the formatter isn't applied on the std lib there :upside_down_face: